packages/common/src/utils.js

```diff
@@ -315,12 +315,16 @@ const sortRoutes = function sortRoutes(routes) {
   return routes
 }
 
-export const createRoutes = function createRoutes(files, srcDir, pagesDir) {
+export const createRoutes = function createRoutes(files, srcDir, pagesDir, typescript = false) {
```

Comment:
@pi0 @kevinmarrec I wonder if we should add `supportedExtension` somewhere under `nuxt/config`.

Reply:
Perhaps under the `router` property in `nuxt.config.js`.

Reply:
@galvez This `createRoutes` is used in `test/unit/utils.test.js` which only test utils, so i don't know if we can pass supportedExtensions here :/.

And i don't know about the `nuxt/config`